packages/wdio-cli/src/index.js

```diff
@@ -1,35 +1,30 @@
+import fs from 'fs'
+import process from 'process'
+import path from 'path'
+
 import yargs from 'yargs'
-import pickBy from 'lodash.pickby'
 
-import { CLI_PARAMS, USAGE } from './config'
 import Launcher from './launcher'
-import runCLI from './run'
+import { handler } from './commands/run'
 
-export const run = () => {
-    let argv = yargs
-        .usage(USAGE)
+export const run = async () => {
+    const argv = yargs
         .commandDir('commands')
+        .demandCommand()
+        .help()
 
     /**
-     * populate cli arguments
+     * if yargs doesn't run a command from the command directory
+     * we verify if a WDIO config file exists in the given "path" parameter
+     * if so, we assume the user ran `wdio path/to/wdio.conf.js` and we execute `wdio run` command
      */
-    for (const param of CLI_PARAMS) {
-        argv = argv.option(param.name, param)
-    }
-
-    const params = pickBy(argv.argv)
+    const params = { ...argv.argv }
+    const wdioConfPath = path.join(process.cwd(), params._[0])
+    const wdioConf = fs.existsSync(wdioConfPath)
 
-    /**
-     * fail execution if more than one wdio config file was specified
-     */
-    if (params._.length > 1) {
-        argv.showHelp()
-        console.error(`More than one config file was specified: ${params._.join(', ')}`) // eslint-disable-line
```

Comment:
Is this still failing if multiple configs are passed in the wdio command?

Reply:
@christian-bromann  Doesn't seem like it is. Currently if you pass two it will just run the first one since we're passing only the first parameter to find the configuration [here](https://github.com/webdriverio/webdriverio/blob/4dace8d54d91d8a147874f6d2fb360692667bda6/packages/wdio-cli/src/index.js#L22).

If you want I can add a check before this and error if it has more than one.